### ISSUE TITLE
Error: Theme base16-onedark does not exist in assets

### ISSUE THREAD

**Issue by luctius** [dustinblackman/oatmeal#10]
When using `cargo install oatmeal`, I get the following error:

```
Oh no! Oatmeal has failed with the following app version and error.

Version: 0.6.1
Commit: VERGEN_IDEMPOTENT_OUTPUT
Error: Theme base16-onedark does not exist in assets
```

**Comment by dustinblackman** [comment]
Good call, thanks for reporting! I'll get it fixed up.

**Comment by dustinblackman** [comment]
Fixed in [v0.7.0](https://github.com/dustinblackman/oatmeal/releases/tag/v0.7.0). Thanks!
